orders/tests.py

```diff
@@ -0,0 +1,83 @@
+from django.utils import timezone
+from .models import Order
+from .models import StatusOptions
+import pytest
+
+EMAIL = 'testuser@gmail.com'
+DOG_OWNER_ID = '123456789'
+DAYCARE_ID = '987654321'
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now()
+TOTAL_PRICE = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def order():
+    return Order(dog_owner_id=DOG_OWNER_ID,
+                 daycare_id=DAYCARE_ID,
+                 starting_date=STARTING_DATE,
+                 finishing_date=FINISHING_DATE,
+                 total_price=TOTAL_PRICE,
+                 status=STATUS
+                 )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self):
+        new_order = Order.create(DOG_OWNER_ID,
+                                 DAYCARE_ID,
+                                 STARTING_DATE,
+                                 FINISHING_DATE,
+                                 TOTAL_PRICE)
+        assert new_order in Order.objects.all()
+
+    def test_delete_order(self, order):
+        order = Order.objects.first()
+        assert order is not None
+
+        order.delete()
+        assert order not in Order.objects.all()
+
+    def test_get_all_orders_by_dog_owner_id(self, order):
+        dog_owner_orders = Order.get_all_dog_owner_orders(order.dog_owner_id)
+
+        print(dog_owner_orders)
+
+    def test_get_all_orders_by_daycare_id(self, order):
+        order.save()
+        daycare_orders = Order.get_all_daycare_orders(order.daycare_id)
+
+        print(daycare_orders)
+
+    def test_get_order_by_order_id(self):
+        order = Order.objects.first()
+        order_id = order.order_id
+
+        order = Order.objects.get(order_id=order_id)
+        assert order is not None
+        print(order)
+
+    def test_approve_order(self, order):
+        order.save()
+        new_status = StatusOptions.Approved
+
+        result = order.update_status(new_status=new_status)
+        assert result != "invalid"
+        order.refresh_from_db()
+        print(f"Approving date: {order.approving_date.__format__('%Y-%m-%d %H:%M:%S')}")
+
+    def test_cancel_order(self, order):
+        order.save()
+        new_status = StatusOptions.Canceled
+
+        result = order.update_status(new_status=new_status)
+        assert result != "invalid"
+        order.refresh_from_db()
+        print(f"Canceling date: {order.canceling_date.__format__('%Y-%m-%d %H:%M:%S')}")
+
+    def test_order_migrate(self):
+        order = Order.objects.get(dog_owner_id='12345')
+
+        assert order in Order.objects.all()
```

Comment:
consider adding the tests:
- status max length (you can use the validators built-in Django as I used in the daycare app
- test what happens if we delete daycare/dogowner user - I think the order should remain anyway in order to see the history of orders anyway